### DISCUSSION TITLE
Intellichlor Salt Generator Output Question

### DISCUSSION THREAD

**Discussion by FlaMike** [tagyoureit/nodejs-poolController#673] [Q&A]
I have a Pentair Intellichlor chlorine generator and an EasyTouch 2.4 controller.  I am running version 7.70 of the njsPC.  The water temperature of the pool is 47 degrees so the salt cell is shut down due to low temperature currently (the red LED indicating "cold water system off" is illuminated).  The salt generator is set to run 8% of the time when the water is warm enough.  The njsPC Dashboard shows salt cell output of 8%, as opposed to 0.

When I go to the EasyTouch panel & look at the salt cell diagnostics output, it says the cell is shut down due to low temperature (I don't know the precise verbiage, but I can provide it if that helps).

Is the EasyTouch reporting the cell output incorrectly or is there something in njsPC or Dashboard that is  displaying the incorrect value?  It's not a huge deal, but I am curious if the reported output is accurate or always reflects the setpoint.

Thanks for your help & fabulous system!

**Comment by rstrouse** [comment]
Upload a Replay for about 1 minute.  Some IntelliChlor models (based on when they were made) report the cold water alarm and very low salt alarm differently.  Perhaps this is a model that we missed.

**Comment by FlaMike** [comment]
file:///home/michael/Downloads/replay.zip

![image](https://user-images.githubusercontent.com/41341956/210079688-d5232d2a-97e1-4716-b9fd-bc20b6cf4a8d.png)

Thank you for your quick response.

Here is a replay of the system after it started up this morning.  This may turn out to be a non-issue; the cell output has gone from zero to 8% and now back to  zero.  The only difference I can think of since yesterday is I uploaded the latest version of Crewski's HACS integration.  He corrected the heater toggle problem I experienced in early November.  There may have been other changes made as well.  Perhaps they fixed the Intellichlor output issue, but I can't imagine how they affected the dashboard program (it currently shows the correct output of 0).

Both the dashboard and HA display a salt concentration of zero, which is incorrect.  It should be about 2950, based on yesterday's readings.  Very strange, indeed!

Thank you for your help!!

**Reply by tagyoureit** [reply]
The replay didn't attach to the thread correctly.

**Reply by FlaMike** [reply]
So sorry.  I perhaps this time will work.  BTW, salt concentration still reads zero.  Shrug.
file:///home/michael/Downloads/replay.zip

**Reply by FlaMike** [reply]
/home/michael/Downloads/replay.zip

**Reply by FlaMike** [reply]
I have tried drag and drop and cut and paste.  I don't think either is working :-(

**Reply by FlaMike** [reply]
[replay.zip](https://github.com/tagyoureit/nodejs-poolController/files/10325445/replay.zip)

**Reply by FlaMike** [reply]
Maybe this time?

**Reply by rstrouse** [reply]
HASS didn't affect the output from the chlorinator this is simply because EasyTouch didn't contact the chlorinator to get its updated status.   From the replay, there is contact and the alarm bit is set from the chlorinator response.

**Reply by rstrouse** [reply]
Also, if there is a cold water alarm the chlorinator will stop reporting the salt level.  What you actually see in ET is not a valid number.

**Reply by FlaMike** [reply]
OK. Thanks for your responses.  I've not seen the salt level reported as zero previously, but I know the water is too cold to generate chlorine, so  I guess it doesn't matter.

The salt cell setpoint and the output are both shown as "unavailable" in HA.  I've not seen that previously, either.  Again, I guess it doesn't matter given the current pool conditions.

I did try doing a restore from this morning's backup & a reboot after that.  No change in the reporting from the Easytouch.  I guess I could reboot it as well, but I'm not certain of the benefit that would result.
